src/main/java/com/bouncestorage/chaoshttpproxy/ChaosHttpProxyHandler.java

```diff
@@ -173,15 +179,23 @@ public void handle(String target, Request baseRequest,
                 clientRequest.header(headerName, headerValue);
             }
 
+            String content = IOUtils.toString(is);
```

Comment:
Can you use Guava `CharStreams.toString` instead of introducing the commons-io depedency?